### DISCUSSION TITLE
能否给出详细的使用步骤

### DISCUSSION THREAD

**Discussion by Apcin** [YunYouJun/valaxy#318] [Q&A]
waline支持作为组件引入到vue3，而我的项目是Vue2，官方npm导入无法兼容Vue2，这款插件好像可以，但是我对文档描述的引入过程不是很了解，能否给出较为具体的步骤或者demo@YunYouJun 
![image](https://github.com/YunYouJun/valaxy/assets/95763618/d5242b50-b59b-45ce-84a4-a2036621512d)

**Comment by YunYouJun** [comment]
valaxy 是一个基于 Vue3 的项目。
